### PR TITLE
Remove pinned dependency

### PR DESCRIPTION
**Pull request by willingc**
Underlying issue was resolved. Pin is no longer necessary.

----
📚 Documentation preview 📚: https://docs-community--102.org.readthedocs.build/en/102/

### CODE REVIEW

**Comment by hugovk** [comment]
The PR https://github.com/executablebooks/MyST-Parser/pull/842 was merged in December, but hasn't been released yet.

The last release was in June: https://github.com/executablebooks/MyST-Parser/releases

**Comment by willingc** [comment]
@hugovk Just realized that. I'm going to update this PR with the current SHA.

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,7 @@
 Sphinx
-myst-parser
+# myst-parser fixed the docutils warnings. Pinning to current SHA until
+# a new release greater than 2.0.0 is made.
+git+https://github.com/executablebooks/MyST-Parser.git@978e845543b5bcb7af0ff89cac9f798cb8c16ab3
```
Comment (ezio-melotti) [inline requirements.txt:4]: Maybe we can ping them to make a new release so that we don't have to pin the SHA?

Comment (willingc) [reply requirements.txt:4]: Left an issue and pinged Chris: https://github.com/executablebooks/MyST-Parser/issues/870

I'm going to merge the pin for now and we can remove after the release.

**Pull request merged**